### ISSUE TITLE
lat/lon deltas aren't working

### ISSUE THREAD

**Issue by bcardarella** [liveview-native/liveview-native-swiftui-mapkit#208]
I have the following:

```heex
    <Map
      bounds:latitude={34.011_286}
      bounds:longitude={-116.166_868}
      bounds:latitude-delta={0.2}
      bounds:longitude-delta={0.2}
      />
```

and it should render this:

![image](https://github.com/liveview-native/liveview-native-swiftui-mapkit/assets/18524/7069a7fc-892e-4730-9de7-220494391492)

but instead I get this:

<img width="559" alt="Screenshot 2024-01-21 at 8 04 21 PM" src="https://github.com/liveview-native/liveview-native-swiftui-mapkit/assets/18524/c3b026e1-3e26-4e4f-9559-aae6e1ec4714">

I believe the location is working as when I change the lat/lon values it will center on the correct location.

When I look at the View Hierarchy I see the deltas present:

<img width="397" alt="Screenshot 2024-01-21 at 8 07 46 PM" src="https://github.com/liveview-native/liveview-native-swiftui-mapkit/assets/18524/a6572eba-3a71-4420-b29d-4f19927b722b">

It's not clear to me why the deltas are not working to zoom the map

**Comment by bcardarella** [comment]
I did render the map correct in SwiftUI with:

```swift
import SwiftUI
import MapKit

struct ContentView: View {
    var body: some View {
        Map(initialPosition: .region(region))
    }

    private var region: MKCoordinateRegion {
            MKCoordinateRegion(
                center: CLLocationCoordinate2D(latitude: 34.011_286, longitude: -116.166_868),
                span: MKCoordinateSpan(latitudeDelta: 0.2, longitudeDelta: 0.2)
            )
        }
}
```

which renders as expected in the first image above. The MapContentView in the View Hierarchy looks different too:

<img width="868" alt="Screenshot 2024-01-21 at 8 28 23 PM" src="https://github.com/liveview-native/liveview-native-swiftui-mapkit/assets/18524/21d5ec6c-440b-4c03-8960-9320a8c225cf">

**Comment by bcardarella** [comment]
This might be helpful, the issue I don't believe is the max/min bounds but the default zooom. See the video of the LVN render: https://www.youtube.com/watch?v=Ip_pDOj931I

the bounds appear to be in place so whatever default Zoom that example I have appears to be doing something different

**Comment by carson-katri** [comment]
Instead of setting the `bounds` you'll need to set the `position`. It needs to be JSON encoded:

```heex
<Map
  position={Jason.encode!(%{ type: :region, center: [34.011_286, -116.166_868], latitude_delta: 0.2, longitude_delta: 0.2 })}
/>
```

**Comment by bcardarella** [comment]
Fixed in #209
